python/paddle/fluid/executor.py

```diff
@@ -522,6 +528,15 @@ def run(self,
                 scope=scope,
                 return_numpy=return_numpy,
                 use_program_cache=use_program_cache)
+        else:
```

Comment:
take a look at L543, and do something similar.

Reply:
Could you tell me more details？

Reply:
Please read the executor.py can try harder to be consistent. I believe you can do it.

```
        if program._is_data_parallel:
            return self._run_parallel(...)
```